test/content/UniqueContentStorageTests.js

```diff
@@ -233,13 +233,13 @@ describe('Unique Content Storage Contract Tests', () => {
 
             var tokenFees2 = await uniqueContentStorage.getMultipleRoyalties(0, 200000);
             expect(tokenFees2[0][0]).to.equal(developerAddress.address);
-            expect(tokenFees2[0][1]).to.equal(ethers.constants.AddressZero);
-            expect(tokenFees2[0][2]).to.equal(ethers.constants.AddressZero);
-            expect(tokenFees2[0][3]).to.equal(ethers.constants.AddressZero);
+            expect(tokenFees2[0][1]).to.equal(undefined);
```

Comment:
Just check whether tokenFees2[0] and tokenFees2[1] length is equal to 1, and then delete the [1] to [3]